src/stores/omniscientLogger.ts

```diff
@@ -12,93 +12,93 @@ export const webrtcStats = new WebRTCStats({ getStatsInterval: 250 })
 export const useOmniscientLoggerStore = defineStore('omniscient-logger', () => {
   const videoStore = useVideoStore()
 
-  // Routine to log the framerate of the video streams
-  const streamsFrameRateHistory = ref<{ [key in string]: number[] }>({})
-  let lastStreamAverageFramerateLog = new Date()
-  const streamAverageFramerateLogDelay = 10000
-  setInterval(() => {
-    Object.keys(videoStore.activeStreams).forEach((streamName) => {
-      if (videoStore.activeStreams[streamName] === undefined) return
```

Comment:
1. what defines whether the stream is active?
2. are the data from the inactive ones being freed?

Reply:
1. All incoming streams are activated by default.
2. Yes

But just to be clear, you can ignore the code changes. I just pushed what was in my staging area to open the PR and start to add the performance stats.